### PR TITLE
immer pinentry. pinentry cleanup. better typing for remote windows

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/actions/json/pinentry.json [hunk 1]
```diff
@@ -1,36 +1,17 @@
 {
-  "prelude": [],
+  "prelude": [ ],
   "actions": {
-    "deleteEntity": {
```
Comment (chrisnojima) [inline shared/actions/json/pinentry.json]: this is a holdover from the entity days

File: shared/pinentry/remote-proxy.desktop.tsx [hunk 2]
```diff
@@ -1,86 +1,44 @@
...
-    submitted: data.submitted,
-    type: data.type,
-    windowComponent: 'pinentry',
-    windowOpts: {height: 210, width: 440},
-    windowParam: String(data.sessionID),
-    windowPositionBottomRight: false,
-    windowTitle: 'Pinentry',
-  })
-)
+export type WireProps = {
```
Comment (chrisnojima) [inline shared/pinentry/remote-proxy.desktop.tsx]: the props we send across the wire. before it was mostly any all over and now we can use this to better tie things together

**Comment by chrisnojima** [comment]
- [ ] after release

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by buoyad** [comment]
If I `keybase login` with this pinentry opens for a moment then closes

**Comment by chrisnojima** [comment]
Did you restart hot server

**Pull request merged**